lona/app.py

```diff
@@ -236,7 +236,7 @@ def parse_command_line(self) -> dict:
 
         parser.add_argument(
             '--debug-mode',
-            choices=['messages', 'views', 'input-events'],
+            choices=['messages', 'views', 'input-events', 'view-events'],
```

Comment:
Just for my understanding. Am I right that only one of this modes can be used in a moment? So I can't debug `messages` and `view-events`, for example.

Reply:
Thats right. If you wan't to see both you can use ``--loggers lona.messages lona.view_events``.